packages/dds/merge-tree/src/client.ts

```diff
@@ -729,7 +728,7 @@ export class Client extends TypedEventEmitter<IClientEvents> {
      * If the position refers to a segment/offset that was removed by some operation between `seqNumberFrom` and
      * the current sequence number, returns undefined.
      */
-    public rebasePositionWithoutSegmentSlide(
+    public rebasePosition(
```

Comment:
hm, maybe we should just delete this API from client entirely, since `findReconnectionPosition` is already exposed so consumers can just call that.